client/getkeys.go

```diff
@@ -11,7 +11,7 @@ var cmdGetKeys = &Command{
 	Long: `
 Get Keys takes version ids returns matching key ids if they exist.
 
-If no version ids, are given it returns all version ids.
+If no version ids are given it returns all version ids.
```

Comment:
, belongs after "given" . do we want to change the language here to refer to key ids rather than version ids? I think the term version id is a bit confusion given each key has a list of versions.

Reply:
yes I think it makes sense to change to `key ids` since we already mentioned it (`returns matching key ids`) on line 12